tests/openkit-shared-tests/Core/Configuration/OpenKitConfigurationTest.cs

```diff
@@ -172,6 +175,25 @@ public void ADefaultConstructedConfigurationUsesStrictTrustManager()
             Assert.That(sslTrustManager, Is.InstanceOf<SSLStrictTrustManager>());
         }
 
+        [Test]
+        public void GetApplicationID()
+        {
+            // given
+            var target = CreateDefaultConfig();
+
+            // then
+            Assert.That(target.ApplicationID, Is.EqualTo("/App_ID%"));
+        }
+
+        [Test]
+        public void getApplicationIDPercentEncodedDoesPropperEncoding()

```

Comment:
test name starting with minor letter

shouldn't this be GetApplicationIDPercentEncodedDoesPropperEncoding